js/plot/util/scales.js

```diff
@@ -86,7 +86,10 @@ var scales = function(opts) {
       if ((!data) || (data.length === 0)) {
         return [];
       }
-      var defaultTicks = opts.bgUnits === 'mg/dL' ? [40, 80, 120, 180, 300] : [2.0, 4.5, 7.0, 10.0, 16.0];
+      var defaultTicks = _.map(_.values(_.omit(opts.bgClasses, 'very-high')), function(n) {
+        return _.get(n, 'boundary');
+      }).sort(function (a, b) { return a - b; });
```

Comment:
So I imagine the fact that the low and very-low overlap on the daily view is going to come up in design review... Have you scheduled one? If not, I think a quick one should be scheduled due to that issue.